### PR TITLE
testnet - cli_wallet command

### PR DESCRIPTION
**Pull request by cedar-book**

### CODE REVIEW

File: docs/development/testnets/public_testnet.rst [hunk 1]
```diff
@@ -75,7 +75,7 @@ As you know, each blockchain starts with a genesis block that definitions are in
 
 The BitShares Blockchain Data Configuration file exists in the blockchain data directory. And the data directory will be created by launching a witness_node. 
 
-At this point, we just want to create a :ref:`configuration file (config.ini)  <bts-config-ini-eg-public-testnet>` to observe the parameters. If you want to use the Public Testnet, you can use the default configuration settings without changes.
+At this point, we just want to create a :ref:`configuration file (cinfig.ini)  <bts-config-ini-eg-public-testnet>` to observe the parameters. If you want to use the Public Testnet, you can use the default configuration settings without changes.
```
Comment (abitmore) [inline docs/development/testnets/public_testnet.rst]: Typo

File: docs/development/testnets/public_testnet.rst [hunk 2]
```diff
@@ -87,7 +87,7 @@ Run the following command and end the process. In Windows, you can use ``[ctl]+c
 2-2. Blockchain Data Directory
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-As a default the BitShares Blockchain data directory ``witness_node_data_dir`` will be created in a current directory.
+As a default the BitShatres Blockchain data directory ``witness_node_data_dir`` will be created in a current directory.
```
Comment (abitmore) [inline docs/development/testnets/public_testnet.rst]: Typo

File: docs/development/testnets/public_testnet.rst [hunk 3]
```diff
@@ -137,7 +137,7 @@ If you want to create own data directory, use ``--data-dir`` parameter and run t
 2-4. Observe the config.ini file
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Although you can use the default :ref:`config.ini <bts-config-ini-eg-public-testnet>` file for the Public Testnet, it's worth to recognize the parameters. 
+Although you can use the default :ref:`cinfig.ini <bts-config-ini-eg-public-testnet>` file for the Public Testnet, it's worth to recognize the parameters.
```
Comment (abitmore) [inline docs/development/testnets/public_testnet.rst]: Typo

**Comment by cedar-book** [comment]
Sorry, and Thank you!!!

**Pull request merged**